scripts/category/naquadah_catalysts_definitions.zs

```diff
@@ -0,0 +1,111 @@
+#loader contenttweaker
+
+//825_Jaded_Vector
+
+import mods.contenttweaker.VanillaFactory;
+import mods.contenttweaker.Item;
+import mods.contenttweaker.Fluid;
+import mods.contenttweaker.Color;
+import mods.contenttweaker.CreativeTab;
+
+val Nucleothurgy = mods.contenttweaker.VanillaFactory.createCreativeTab("Nucleothurgy", <item:contenttweaker.naquadah_catalyst_grade_1>);
+Nucleothurgy.unlocalizedName = "Nucleothurgy";
+Nucleothurgy.register();
+
+//catlyst definitions
+//Grade 1 (astral)
+var NRG1 = VanillaFactory.createItem("naquadah_resevoir_grade_1");
+NRG1.maxStackSize = 1;
+NRG1.maxDamage = 10000;
+NRG1.rarity = "rare";
+NRG1.creativeTab = Nucleothurgy;
+NRG1.register();
+var NCG1 = VanillaFactory.createItem("naquadah_catalyst_grade_1");
+NCG1.creativeTab = Nucleothurgy;
+NCG1.register();
+
+//Grade 2 (botanic)
+var NRG2 = VanillaFactory.createItem("naquadah_resevoir_grade_2");
+NRG2.maxStackSize = 1;
+NRG2.maxDamage = 15000;
+NRG2.rarity = "rare";
+NRG2.creativeTab = Nucleothurgy;
+NRG2.register();
+var NCG2 = VanillaFactory.createItem("naquadah_catalyst_grade_2");
+NCG2.creativeTab = Nucleothurgy;
+NCG2.register();
+
+//Grade 3 (alchemic)
+var NRG3 = VanillaFactory.createItem("naquadah_resevoir_grade_3");
+NRG3.maxStackSize = 1;
+NRG3.maxDamage = 20000;
+NRG3.rarity = "rare";
+NRG3.creativeTab = Nucleothurgy;
+NRG3.register();
+var NCG3 = VanillaFactory.createItem("naquadah_catalyst_grade_3");
+NCG3.creativeTab = Nucleothurgy;
+NCG3.register();
+
+//Grade 4 (thaumic)
+var NRG4 = VanillaFactory.createItem("naquadah_resevoir_grade_4");
+NRG4.maxStackSize = 1;
+NRG4.maxDamage = 72000;

```

Comment:
This is potentially buggy configuration.
Most mods treat `Damage` item value as `32,767` at max. This means, that if other mods would interact with this item, they could produce unexpected result, such as
- Negative durability
- Infinite durability
- Breaking item


Make sure, that repairing this item not breaking it.
Make sure that consuming durability of this item between values 32,769 - 32,765 actually work.